### PR TITLE
Refactor schedule displaying related methods into ScheduleUi class

### PR DESCRIPTION
**Pull request by johan9se**

### CODE REVIEW

File: src/main/java/seedu/lifeasier/commands/DisplayScheduleCommand.java [hunk 1]
```diff
@@ -25,77 +21,14 @@ public DisplayScheduleCommand(String toDisplay) {
...
+            scheduleUi.displayWeekSchedule(currDate, tasks);
         } else {
-            if (getTaskCountForToday(tasks, currDate) == 0) {
+            if (scheduleUi.getTaskCountForToday(tasks, currDate) == 0) {
                 System.out.println("You have nothing on for today!");
             } else {
                 System.out.println("Here is your schedule for today:");
-                displayDaySchedule(currDate, tasks);
+                scheduleUi.displayDaySchedule(currDate, tasks);
             }
```
Comment (daniellimzj) [inline src/main/java/seedu/lifeasier/commands/DisplayScheduleCommand.java]: You could consider placing this section before the case for when there are no tasks in the day. Something to do with making the happy path prominent:
https://nus-cs2113-ay2021s1.github.io/website/se-book-adapted/chapters/codeQuality.html#make-the-happy-path-prominent

File: src/main/java/seedu/lifeasier/commands/DisplayScheduleCommand.java [hunk 2]
```diff
@@ -2,20 +2,16 @@
 
 import seedu.lifeasier.storage.FileStorage;
 import seedu.lifeasier.tasks.TaskList;
-import seedu.lifeasier.tasks.Task;
-import seedu.lifeasier.tasks.Deadline;
-import seedu.lifeasier.tasks.Event;
-import seedu.lifeasier.tasks.Lesson;
 import seedu.lifeasier.ui.Ui;
```
Comment (daniellimzj) [inline src/main/java/seedu/lifeasier/commands/DisplayScheduleCommand.java]: Should this import be removed if we are no longer using the Ui class?

**Review by daniellimzj — COMMENTED** [review]
Just 2 very minor things!

File: src/main/java/seedu/lifeasier/commands/DisplayScheduleCommand.java (hunk 1 above)
Comment (johan9se) [reply src/main/java/seedu/lifeasier/commands/DisplayScheduleCommand.java]: Oh yes! Thanks for that reminder, I've changed it accordingly!

File: src/main/java/seedu/lifeasier/commands/DisplayScheduleCommand.java (hunk 2 above)
Comment (johan9se) [reply src/main/java/seedu/lifeasier/commands/DisplayScheduleCommand.java]: But isn't Ui being passed into the execute method of all Command classes?

Comment (daniellimzj) [reply src/main/java/seedu/lifeasier/commands/DisplayScheduleCommand.java]: Oh yes I forgot about that. My bad!

**Pull request merged**